style.css

```diff
@@ -103,19 +109,20 @@ main {
 }
 
 .work-experience-part-two-description {
-    width: 39%;
+    width: 300px;
     font-size: 20px;
 }
 
 .work-experience-part-two-list {
-    width: 100%;
+    width: 104%;
```

Comment:
Setting an element's width to more than 100% might push other elements out of their expected positions or cause unexpected wrapping or scaling. I'd advise against it